ep_gui/resources/lang/ep_gui.en-US.rc

```diff
@@ -362,16 +362,16 @@ BEGIN
     IDS_OTHER_PWRBTNACTION_2    "Shut down (default)"
     IDS_OTHER_PWRBTNACTION_4    "Restart"
     IDS_OTHER_NOREDIRECT        "Prevent the following Control Panel links from being redirected to the Settings app:"
-    IDS_OTHER_NOREDIRECT_SYSTEM "System / About page"
-    IDS_OTHER_NOREDIRECT_PROGRAMS "Programs and Features"
-    IDS_OTHER_NOREDIRECT_DATETIME "Adjust date/time"
-    IDS_OTHER_NOREDIRECT_TRAYICONS "Customize notification icons"
+    IDS_OTHER_NOREDIRECT_SYSTEM "System/About page"
+    IDS_OTHER_NOREDIRECT_PROGRAMS "Programs && Features"
+    IDS_OTHER_NOREDIRECT_DATETIME "Adjust Date && Time"

```

Comment:
This should be changed to "Adjust Date and Time".

Reply:
For the same reasons I validate the proposal